boards/riscv/icev_wireless/icev_wireless.dts

```diff
@@ -74,7 +75,27 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 	pinctrl-0 = <&spim2_default>;
-	pinctrl-names = "default";
+	pinctrl-1 = <&spim2_sleep>;
+	pinctrl-2 = <&spim2_cs_as_gpio>;
+	pinctrl-names = "default", "sleep", "cs-as-gpio";
+
+	cs-gpios = <&gpio0 6 GPIO_ACTIVE_LOW>;
```

Comment:
see my previous comment regarding props

Reply:
GPIO_ACTIVE_LOW is present in almost all cs-gpios properties for spi controllers